frontend/packages/kubevirt-plugin/src/components/vm-disks/table-filters.ts

```diff
@@ -0,0 +1,44 @@
+import * as _ from 'lodash';
+import { RowFilter } from '@console/internal/components/filter-toolbar';
+import { DiskType } from '../../constants/vm/storage';
+
+const sourceReducer = (obj) => {
+  const source = obj?.source;
+  const type = obj?.metadata?.type;
+
+  if (type === DiskType.CDROM) {
+    return 'cdrom';
+  }
+
+  if (['Config Map', 'Secret', 'Service Account'].includes(source)) {
+    return 'env';
+  }
+
+  return 'disk';
+};
+
+export const diskSourceFilter: RowFilter = {
+  filterGroupName: 'Category',
+  type: 'disks',
+  reducer: sourceReducer,
+  items: [
+    {
+      id: 'disk',
+      title: 'Disks',
```

Comment:
these could be taken from DiskType.toString

Reply:
+1, only filtering by type now, removed this part of the code.